src/main/resources/sql/create.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE IF NOT EXISTS "shop" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255)        NOT NULL,
+  "address" VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "customer" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255) NOT NULL,
+  "surname" VARCHAR(255) NOT NULL,
+  "age"     INT2         NOT NULL,
```

Comment:
Please explain what is INT2, and why you use it.

Reply:
2 bytes integer, the same as smallint. I use it because there is no 1-byte integer in the standard distribution of Postgres and I took the smallest. The type names int2, int4, and int8 are extensions, I saw their usage in some video... Now I see that it's not good. I will change INT2 on SMALLINT. Thanks!

Reply:
Actually in postgres int2 and smallint are the same. But here are all sql types in the table https://goo.gl/Jd8gqU